tests/test_pe_reports.py

```diff
@@ -105,3 +105,29 @@ def test_reports_bad_log_level():
         except SystemExit as sys_exit:
             return_code = sys_exit.code
         assert return_code == 1, "main() should exit with error"
+
+
+@pytest.fixture
+def client():
+    """Create client to test flask application."""
+    flask_app.config.update({"TESTING": True})
+
+    with flask_app.test_client() as client:
+        yield client
+
+
+# TODO: Increase flask UI testing to test Cyber Six Gill API responses. The
+#   current state of the CSG API times out randomly.
+#   See https://github.com/cisagov/pe-reports/issues/213
+def test_home_page(client):
+    """Test flask home.html is avalible and verify the correct page."""
+    resp = client.get("/")
+    assert resp.status_code == 200
+    assert b"Home" in resp.data
+
+
+def test_stakeholder_page(client):
+    """Test flask home_stakeholder.html and verify the correct page."""
```

Comment:
Mirror previous suggestion:
```suggestion
    """Test flask home_stakeholder.html is available and verify a string on the page."""
```